RecoTracker/LSTCore/src/LSTESData.cc

```diff
@@ -10,25 +10,25 @@
 
 namespace {
   std::string geometryDataDir() {
-    const char* path_lst_base = std::getenv("LST_BASE");
+    std::string path_str, path;
     const char* path_tracklooperdir = std::getenv("TRACKLOOPERDIR");
-    std::string path_str;
-    if (path_lst_base != nullptr) {
-      path_str = path_lst_base;
-    } else if (path_tracklooperdir != nullptr) {
+    std::stringstream search_path;
+    search_path << std::getenv("CMSSW_SEARCH_PATH");
```

Comment:
I changed the initialization here since the IB was complaining about a `nullptr`.